README.md

```diff
@@ -126,27 +126,33 @@ Usage:
+      --chown string          chown the ouput directory to match the specified UID:GID
+      --output string         artifact output directory (default ".")
+      --progress string       type of progress bar to use (e.g. verbose,term) (default "auto")
+      --rootfs string         Root filesystem type. If not given, the default configured in the source container image is used.
+      --target-arch string    build for the given target architecture (experimental)
+      --tls-verify            require HTTPS and verify certificates when contacting registries (default true)
+      --type stringArray      image types to build [ami, anaconda-iso, gce, iso, qcow2, raw, vhd, vmdk] (default [qcow2])
+      --version               version for build
 
 Global Flags:
+      --log-level string   logging level (debug, info, error); default error
   -v, --verbose            Switch to verbose mode
 ```
 
 ### Detailed description of optional flags
 
 | Argument          | Description                                                                                               | Default Value |
 |-------------------|-----------------------------------------------------------------------------------------------------------|:-------------:|
-| **--chown**       | chown the output directory to match the specified UID:GID                                                 |       ❌      |
-| **--progress**   | Show progress in the given format, supported: verbose,term,debug. If empty it is auto-detected               |   `auto`      |
+| --chown           | chown the output directory to match the specified UID:GID                                                 |       ❌      |
+| --output          | output the artifact into the given output directory                                                       |      `.`      |
+| --progress        | Show progress in the given format, supported: verbose,term,debug. If empty it is auto-detected            |     `auto`    |
 | **--rootfs**      | Root filesystem type. Overrides the default from the source container. Supported values: ext4, xfs, btrfs |       ❌      |
-| **--tls-verify**  | Require HTTPS and verify certificates when contacting registries                                          |    `true`     |
-| **--type**        | [Image type](#-image-types) to build                                                                      |    `qcow2`    |
-| **--target-arch** | [Target arch](#-target-architecture) to build                                                             |       ❌      |
-| **--verbose**     | Switch output/progress to verbose mode    |   `false`   |
+| --tls-verify      | Require HTTPS and verify certificates when contacting registries                                          |     `true`    |
+| **--type**        | [Image type](#-image-types) to build (can be passed multiple times)                                       |     `qcow2`   |
```

Comment:
There's a mix of information on how `--type` should be passed in.  One place says it can be passed in multiple times (`--type qcow2 --type raw`), and another says to use a string array (`--type [qcow2,raw]`).  

Even if both are supported, I think it would be good to standardise on which is recommended to avoid confusion.

Reply:
Let's start here and turn the above comment into an issue (I'll click the button).